src/types/profile.js

```diff
@@ -95,7 +95,14 @@ export type StackTable = {|
  * are indices into other tables.
  */
 export type SamplesTable = {|
-  responsiveness: Array<?Milliseconds>,
+  // Responsiveness is the older version of eventDelay. It injects events every 16ms.
```

Comment:
If the responsiveness becomes truly optional, then these two comments will need to be updated.